compiler/bindings/python/IREECompilerDialectsModule.cpp

```diff
@@ -518,6 +518,36 @@ NB_MODULE(_ireeCompilerDialects, m) {
             return std::nullopt;
           });
 
+  //===-------------------------------------------------------------------===//
+  // Binding to utility function getSingleSubgroupLayout
+  //===-------------------------------------------------------------------===//
+  py::class_<ireeGPUMMASingleSubgroupLayout>(iree_gpu_module,
+                                             "GPUMMASingleSubgroupLayout")
+      .def_prop_ro(
+          "outer",
+          [](const ireeGPUMMASingleSubgroupLayout &self) { return self.outer; })
+      .def_prop_ro("thread",
+                   [](const ireeGPUMMASingleSubgroupLayout &self) {
+                     return self.thread;
+                   })
+      .def_prop_ro("tstrides",
+                   [](const ireeGPUMMASingleSubgroupLayout &self) {
+                     return self.tstrides;
+                   })
+      .def_prop_ro("element", [](const ireeGPUMMASingleSubgroupLayout &self) {
+        return self.element;
+      });
+
+  iree_gpu_module.def(
+      "get_single_subgroup_layout",
+      [](MlirAttribute attr, int fragment) {
+        return ireeGPUGetSingleSubgroupLayout(attr, fragment);
+      },
+      "Returns the single subgroup layout (element, thread, outer, "
+      "tstrides) "
+      "for a given MMA or VirtualMMA intrinsic and fragment.",
```

Comment:
nit: this string is broken in a weird way -- could you reflow this? I'd expect something like
```suggestion
      "tstrides) for a given MMA or VirtualMMA intrinsic and "
      "fragment.",
```